org.openhab.binding.zigbee/src/test/java/org/openhab/binding/zigbee/internal/ZigBeeThingTypeMatcherTest.java

```diff
@@ -0,0 +1,93 @@
+package org.openhab.binding.zigbee.internal;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.type.ThingType;
+import org.eclipse.smarthome.core.thing.type.ThingTypeBuilder;
+import org.eclipse.smarthome.core.thing.type.ThingTypeRegistry;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.openhab.binding.zigbee.ZigBeeBindingConstants;
+
+/**
+ *
+ * @author Chris Jackson - Initial Implementation
+ *
+ */
+public class ZigBeeThingTypeMatcherTest {
+    private final String THING_MATCHER_PROPERTY = "typeMatcher";
+
+    @Ignore
```

Comment:
Currently I have problems running tests requiring Mockito - I'm hoping that this will be resolved soon as I know there are moves afoot to fix the dependancy issues.

Reply:
Ah, then I understand better. I just had a look at the issue, and think that it is caused by the use of `ArgumentMatchers`, instead of using the `Mockito` class (which extends `ArgumentMatchers`, and probably does some magic to make it work correctly in the Mockito context).

If one replaces the argument matcher in the test as follows, then the test should run without the Mockito issue:
```
Mockito.when(registry.getThingTypes(Mockito.anyString())).thenReturn(things);
```